graders/c/requirements.txt

```diff
@@ -1,5 +1,5 @@
-clang==18.1.8 # Major version must match the version of clang installed in the Dockerfile
-matplotlib==3.10.1
+clang==20.1.5 # Major version must match the version of clang installed in the Dockerfile
```

Comment:
Please no, check the comment.

Reply:
🙃 yes of course, I should have checked `pur`'s output...